src/web/components/Onwards/Carousel/Carousel.tsx

```diff
@@ -161,28 +165,42 @@ const dotsStyle = css`
     }
 `;
 
-const dotStyle = (index: number) => css`
+const dotStyle = (index: number, totalStories: number) => css`
+    cursor: pointer;
     display: inline-block;
     height: ${space[3]}px;
     width: ${space[3]}px;
     background-color: ${palette.neutral[93]};
-    border-radius: 50%;
+    border-radius: 100%;
+    border: 0 none;
+    padding: 0;
     margin-right: ${space[1]}px;
 
+    &:hover,
+    &:focus {
+        background-color: ${palette.neutral[86]};
+        outline: none;
+    }
+
     /* This is a bit of a hack for the test, while we think of better UX here.
     It's very fragile to things like carousel item count.*/
```

Comment:
Is this comment still true? Will this css still work well if `totalStories` is 1 or 20?